dbt/adapters/greenplum/impl.py

```diff
@@ -4,3 +4,6 @@
 
 class GreenplumAdapter(PostgresAdapter):
     ConnectionManager = GreenplumConnectionManager
+
+    def valid_incremental_strategies(self):
+        return ["append", "delete+insert", "truncate+insert"]
```

Comment:
Are "delete+insert" and "truncate+insert" incremental strategies or table?

Reply:
both incremental: 
- https://github.com/dbt-labs/dbt-core/blob/0d645c227f9e84116ee03c048bad22fce4fcc2b3/plugins/postgres/dbt/adapters/postgres/impl.py#L139
- https://docs.getdbt.com/reference/resource-configs/postgres-configs#incremental-materialization-strategies